lib/functions/ProofUtil.ts

```diff
@@ -7,26 +20,21 @@ import { BAD_PARAMS, JWS_NOT_VALID, JWTHeader, JWTPayload, JWTSignerArgs, ProofO
  *         - jwtSignerCallback: function to sign the proof
```

Comment:
Doc is not in line with implementation anymore

Reply:
changed the doc